lib/features/email/presentation/controller/single_email_controller.dart

```diff
@@ -1175,11 +1175,16 @@ class SingleEmailController extends BaseController with AppLoaderMixin {
     consumeState(_parseCalendarEventInteractor!.execute(
       accountId,
       blobIds,
-      TransformConfiguration.fromTextTransformers(const [
-        SanitizeAutolinkUnescapeHtmlTransformer(),
-        StandardizeHtmlSanitizingTransformers(),
-        NewLineTransformer(),
-      ])
+      TransformConfiguration.create(
+        customTextTransformers: const [
+          SanitizeAutolinkUnescapeHtmlTransformer(),
+          StandardizeHtmlSanitizingTransformers(),
+          NewLineTransformer(),
+        ],
+        customDomTransformers: [
+          SanitizeHyperLinkTagInHtmlTransformer(),
```

Comment:
no need `useTooltip: PlatformInfo.isWeb`?

Reply:
Yes, we used a different method to display the tooltip.